src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,39 @@
 package core.basesyntax;
 
 public class MainApp {
+    private final Machine truck = new Truck();
 
+    private final Machine excavator = new Excavator();
+
+    private final Machine bulldozer = new Bulldozer();
+
+    private final Machine[] machines = {getTruck(), getBulldozer(), getExcavator()};
```

Comment:
The array `machines` should not be initialized with specific getters. Instead, directly use the private fields `truck`, `bulldozer`, `excavator` since they are within the same class.